chia/cmds/init_funcs.py

```diff
@@ -270,6 +296,14 @@ def generate_ssl_for_nodes(ssl_dir: Path, ca_crt: bytes, ca_key: bytes, private:
             prefix = "public"
         key_path = node_dir / f"{prefix}_{node_name}.key"
         crt_path = node_dir / f"{prefix}_{node_name}.crt"
+        if node_certs_and_keys is not None:
```

Comment:
if feels a bit like this function really are two functions glued together. I suppose `crt_path` and `key_path` are shared between the two parts